src/components/ResourceList/ResourceList.tsx

```diff
@@ -323,20 +323,22 @@ export class ResourceList extends React.Component<CombinedProps, State> {
   }
 
   componentDidMount() {
-    this.setState(
-      {
-        listNode: this.listRef.current,
-      },
-      () => {
-        if (this.props.loading) {
-          this.setLoadingPosition();
-        }
-      },
-    );
+    this.forceUpdate();
+    if (this.props.loading) {
+      this.setLoadingPosition();
+    }
   }
 
-  componentDidUpdate(prevProps: Props) {
-    if (this.props.loading && !prevProps.loading) {
+  componentDidUpdate({loading: prevLoading, items: prevItems}: Props) {
+    if (
+      this.listRef.current &&
+      this.itemsExist() &&
```

Comment:
Curious why not just compared the props?

Reply:
They are for the edge case where items begin as undefined and then are supplied